tests/dummy/app/routes/infinity-scrollable-scrollevent.js

```diff
@@ -0,0 +1,4 @@
+import ScrollableRoute from './infinity-scrollable';
+
+export default ScrollableRoute.extend({
```

Comment:
Don't necessarily like this, but to avoid duplication and allow for deterministic tests, I separated out each InfinityObserver, rAF, and scrolEvent options in the lib to three separate routes with a scrollable area.